src/SFA.DAS.AANHub.Data/Repositories/ApprenticesWriteRepository.cs

```diff
@@ -12,5 +12,17 @@ internal class ApprenticesWriteRepository : IApprenticesWriteRepository
         public ApprenticesWriteRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
 
         public void Create(Apprentice apprentice) => _aanDataContext.Apprentices.Add(apprentice);
+
+        public async Task<Apprentice> Patch(Apprentice apprenticeEntity)
+        {
+
+            var apprentice = await _aanDataContext
+            .Apprentices
+            .FindAsync(apprenticeEntity.MemberId);
+
+            apprentice!.Email = apprenticeEntity.Email;
```

Comment:
I don't think email is an optional field in the architecture, so it probably doesn't need to be in the classes

Reply:
This section has been re-written and may need to be reviewed again